lib/pascal.rb

```diff
@@ -1,36 +1,36 @@
+class PascalTriangle
+  attr_reader :first_row, :second_row, :space
+  def initialize
+    @first_row = [1]
+    @second_row = [1,1]
+    @space = " "
+  end
+  def draw(height)
+    num_spaces = height
+    for row in (1..height)
+      print_row(num_spaces, row)
+      num_spaces -= 1
+    end
+  end
 
-class Pascal
-
-
-  def nth_row(row_num=1)
-    row = [1,1]
-    new_row= []
+  private
+  def print_row(num_spaces, row)
+    puts "#{@space*num_spaces}#{nth_row(row).join(" ")}"
```

Comment:
When you added `attr_reader :first_row, :second_row, :space
 ` you actually got `first_row, second_row, space` methods. Those are getter methods. You don't need to use the `@space` variable here you should use the `space` method instead.